compliance/evidence.py

```diff
@@ -66,11 +68,21 @@ def __init__(self, name, category, ttl=DAY, description='', **kwargs):
 
     @classmethod
     def from_evidence(cls, evidence):
+        kwargs = {}
+        for content_flag in CONTENT_FLAGS:
+            if hasattr(evidence, content_flag):
+                kwargs[content_flag] = eval(f'evidence.{content_flag}')
```

Comment:
I would use `getattr(evidence, content_flag)`. In fact, do you really need the if at 73? It seems like they will always have those 2 attributes: `binary_content` and `filtered_content`.

Reply:
And thinking about it, perhaps a `content_type` with `'binary'` or `'filtered'` would be more extensible in the future than one attribute per type. But it should be fine for now.

Reply:
I'm using the `if` because the from_evidence class method is technically applicable to all types of evidence while these flags ATM are only for RawEvidence.  I suppose it wouldn't matter if we passed the kwargs in to other evidence types because they would just ignore them but I wanted to be careful so that I don't introduce something that could crop up later on that may cause us problems.

I did change the `eval` to a `getattr` good call there.  Thanks!